src/tensorflow_dataset_steps/README.md

```diff
@@ -0,0 +1,3 @@
+# Tensorflow Datasets Steps
+These are steps for loading small NumPy datasets from
```

Comment:
Not sure about "small NumPy" datasets, e.g., FashionMNIST is 60K images

Reply:
Ah lol just noticed "small NumPy datasets" is the wording in the Keras API docs as well, nevermind then 😆